corehq/apps/hqadmin/management/commands/clone_domain.py

```diff
@@ -187,11 +187,23 @@ def copy_ucr_datasources(self, new_domain, old_domain):
 
     def save_couch_copy(self, doc, new_domain=None):
         old_id = doc._id
-        del doc._id
+        doc._id = doc.get_db().server.next_uuid()
```

Comment:
no biggie but is this better than `uuid.uuid4().hex`? i really don't like the cloudant uuids that look very similar.